source/_docs/energy.markdown

```diff
@@ -21,3 +21,22 @@ Home Assistant is an open platform and so home energy management is not restrict
 If you have a sensor that returns instantaneous power readings (W or kW), then to add a sensor that returns energy usage or generation (kWh) refer to [Riemann sum integral integration](/integrations/integration/#energy)
 
 <img src='/images/docs/energy/energy-overview.png' alt='Visual representation of how all different energy forms relate.' style='border: 0;box-shadow: none;'>
+
+## Troubleshooting missing entities 
+
+### Condition
+
+You are trying to add a sensor to the energy dashboard, but it does not appear in the selection list. 
+
+### Remedy
+
+To find out why the sensor is not showing, check the following points:
+- The sensor must have the appropriate attributes. Check your entity attributes in **{% my developer_states title="Developer Tools > States" %}** to confirm the following:
+  - `device_class` must be `energy` for electricity grid, solar, or battery categories. It must be `gas` for gas, or `water` for water.
+  - `state_class` must be `total` or `total_increasing`.
+  - The sensor must have an appropriate `unit_of_measurement`. See the help text for each category to see which units are accepted. Units containing an exponent must match superscript characters exactly as defined, e.g. `m³` is accepted, `m3` is not.
```

Comment:
If this is the case, it should be reported.

Reply:
If an unit is not correct, it should be reported with the integration author.

Reply:
I am thinking more about templates here where users specify the unit_of_measurement as part of the template, not specifically integration provided sensor. I would expect that integrations would have this correct.

Reply:
This is not the template documention, it will be highly confusing if we adjust general documentation to template specifics.

Reply:
Ok, though I'm not adding any language specific to templates, I'm just explaining the reason why I mentioned it. 

All I'm in this section is to check if the uom is correct. 

I'm not understanding what your suggestion is here to change? Are you requesting to strike this entire sentence? 

~~Units containing an exponent must match superscript characters exactly as defined, e.g. `m³` is accepted, `m3` is not.~~

Reply:
No, I'm saying we should provide information for the user on what to do when it is incorrect. 

Right now, this writes what to check. So let's assume the user discovers the UoM is indeed incorrect: What should they do? -> Report an issue.

Reply:
This is already in the next line below this section:

> If any of the attributes are not correct, please open an issue against the integration that provides your sensor